test/unit/gateways/ixopay_test.rb

```diff
@@ -24,12 +24,10 @@ def setup
   end
 
   def test_successful_purchase
-    @gateway.expects(:ssl_post).returns(successful_purchase_response)
```

Comment:
This is cleanup that I spotted while working on the ```refund``` fix. We don't need to stub here since stubbing is handled by ```stub_comms```.